### DISCUSSION TITLE
Fan Control causes stuttering in games when monitoring Nvidia GPU using newer drivers - can replicate 100%

### DISCUSSION THREAD

**Discussion by Globespy** [Rem0o/FanControl.Releases#2198] [General]
In relation to the unexplained stutters/frame drops in many games when using the latest Nvidia Drivers, I just tested something and think that I've figured out the culprit(s).

There are two culprits to these stutters in newer drivers, Fan Control and HWInfo64.

Fan Control is a fantastic app, however when it's not running there are ZERO frame stutters. Fans can be controlled through most motherboard BIOS's, albeit not as 'pretty' as Fan Control they do the same thing. 
I have tried setting up Fan Control to only use the Motherboard/CPU sensors (uncheck everything else), and that seems to stop the stuttering, if you still want to use the app.

HWInfo64 will also cause stutters when running with these newer drivers when you are monitoring anything on the GPU, and to be honest it shouldn't be running other than when you are dialing in a new system, or if you are trying to identify issues like heat/voltage when troubleshooting.

MSI Afterburner does not seem to cause any issues, as long as you are not monitoring temps, clock speeds etc. in the monitoring section within settings.

It's probably still a problem with the Nvidia driver/Windows, since the stuttering does not happen with the older drivers like 537.58, but the performance is a little better with these new drivers, so worth a try if you don't have some outlier case for needing to have these other apps open/monitoring GPU stats.

For reference, I tested these scenarios on the latest 546.33 Studio driver, I always prefer Studio drivers as they provide the same gaming performance, and are much more thoroughly tested since they are used by professionals for productivity.

Additionally, it seems that Fan Control is known to cause games to hang for split seconds on Gigabyte motherboards, but I'm guessing other boards have the same issues.

**Comment by Rem0o** [comment]
Use NvApiWrapper instead of the GPU section of LibreHardwareMonitor in sensors settings.

**Reply by Globespy** [reply]
Thanks! I found that this worked. But had another thing I noticed and created a new post about yesterday, despite being back on much older drivers that don't cause the Windows slowdowns that some of the newer drivers have also been causing.
